tests/config/wdio.config.ts

```diff
@@ -12,6 +12,9 @@ export const config: Options.Testrunner = {
     },
   },
   specs: ["../features/**/*.feature"],
+  specFileRetries: 2,
+  specFileRetriesDelay: 3,
+  specFileRetriesDeferred: true,
```

Comment:
I think means we will retry failed files twice, but at the end of all the tests, right?

If so I think it makes sense to retry immediately instead in case your first ran test has a bug and you'll have to wait for all the other tests to run before a pipeline fails.

Reply:
@iFergal you're right, I've changed that, commit is pushed